src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -213,12 +214,11 @@ ValidatorSite::onTimer (
     std::size_t siteIdx,
     error_code const& ec)
 {
-    if (ec == boost::asio::error::operation_aborted)
-        return;
```

Comment:
I agree that we should just return for `operation_aborted`. The only place we `cancel` is in `stop`, but because that is messing with the `stopping_` variable, I don't think we reliably call `onSiteFetch` in this case.

Reply:
We only ever really want to stop the timer if `stopping_` is true. I believe that the code, as is, accurately does this.

But I’m happy to add the extra check for `operation_aborted` if you think it’s necessary.

Reply:
You're right, actually. We do need to treat `operation_aborted` differently. Pushed a `[FOLD]` commit to address this.